docs/_posts/2015-09-14-community-roundup-27.md

```diff
@@ -48,10 +48,6 @@ Interact with this [visual tour of Relay's architecture](http://sgwilym.github.i
 
 Sam has already launched a product that leverages Relay's data-fetching, optimistic responses, pagination, and mutations &ndash; all atop a Ruby GraphQL server: [new.comique.co](http://new.comique.co/)
 
-## Prototyping in the browser
-
-I ([steveluscher](https://github.com/steveluscher)) whipped up [a prototyping tool](https://facebook.github.io/relay/prototyping/playground.html) that you can use to hack on a schema and a React/Relay app, from the comfort of your browser. Thanks to Jimmy Jia ([taion](https://github.com/taion)) for supplying the local-only network layer, [relay-local-schema](https://github.com/relay-tools/relay-local-schema).
```

Comment:
According to https://github.com/facebook/relay/pull/1965, Relay Playground has been removed. I think this section should be removed as well as https://facebook.github.io/relay/prototyping/playground.html leads to a 404.